bool using_keyboard
---
string response
